cfme/tests/infrastructure/test_vm_power_control.py

```diff
@@ -267,6 +273,7 @@ def test_power_on(self, appliance, testing_vm, verify_vm_stopped, soft_assert):
         soft_assert(
             testing_vm.provider.mgmt.is_vm_running(testing_vm.name), "vm not running")
 
+    @pytest.mark.rhv3
```

Comment:
If we have this test_suspend in T3, then probably all the other suspends should go there as well

Reply:
test_start_from_suspend is marked as T1, and cover suspend functionality as well.
Therefore I marked test_suspend in T3.
The test_suspend_vm_rest (as well as all other vm power control via rest tests) is marked as T2.